### PR TITLE
add azure pipeline

### PR DESCRIPTION
**Pull request by bpasero**
@lszomoru

### CODE REVIEW

**Comment by lszomoru** [comment]
@bpasero, no build/test steps?

**Comment by bpasero** [comment]
@lszomoru yeah I was about to ask, there are no dependencies and no tests for this module actually

**Comment by lszomoru** [comment]
I see. I suggest that you also set `testPlatforms: []` so that only the packaging job is executed.

File: .azure-pipelines/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - name: publishPackage
+    displayName: 🚀 Publish @vscode/sudo-prompt
+    type: boolean
+    default: false
+
+extends:
+  template: azure-pipelines/npm-package/pipeline.yml@templates
+  parameters:
+    npmPackages:
+      - name: @vscode/sudo-prompt
```
Comment (lszomoru) [inline .azure-pipelines/pipeline.yml]: Also please update the name to `sudo-prompt`. Thanks!

Comment (bpasero) [reply .azure-pipelines/pipeline.yml]: Done

**Review by lszomoru — APPROVED** [review]

**Pull request merged**